src/node_crypto.cc

```diff
@@ -2883,11 +2890,13 @@ void CipherBase::SetAuthTag(const FunctionCallbackInfo<Value>& args) {
                 cipher->auth_tag_len_ == tag_len) &&
                IsValidGCMTagLength(tag_len);
   } else {
+#ifndef OPENSSL_NO_OCB
     // At this point, the tag length is already known and must match the
     // length of the given authentication tag.
     CHECK(mode == EVP_CIPH_CCM_MODE || mode == EVP_CIPH_OCB_MODE);
     CHECK_NE(cipher->auth_tag_len_, kNoAuthTagLength);
     is_valid = cipher->auth_tag_len_ == tag_len;
```

Comment:
This might be a dumb question not knowing a lot about the problem domain, but: This is also being skipped for `EVP_CIPH_CCM_MODE` now … is that okay?

Reply:
I would say just omit if the `|| mode == EVP_CIPH_OCB_MODE` if no OCB support.

Reply:
@mscdex that sounds like a good plan, i'll make those changes in the morning!